### PR TITLE
168 update

### PR DESCRIPTION
**Pull request by oskarth**
Misc rough updates for preMVP and some context for kick off call, also basic instructions on STT testing. And make idea render at http://ideas.status.im/.

### CODE REVIEW

File: ideas/168-paid-master-nodes/stt-payment-testing.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Getting and sending STT, proof of payment
+
+Some notes for initial payment MVP and testing thereof.
+
+## How do I get STT?
+
+- Be on Ropsten
+- Advanced settings -> Development mode
+- Chat with console and use `/faucet <StatusFaucet>` to get ETH
+- > For STT simply send a transaction (can be 0 ether) with Gas limit 105000 (you need to set the value manually!) to the following address: 0x34358C45FbA99ef9b78cB501584E8cBFa6f85Cef (https://wiki.status.im/Testing_FAQ_(core_team))
```
Comment (jpbowen) [inline ideas/168-paid-master-nodes/stt-payment-testing.md]: Is this address the current default? If not, can we make it one in faucet?

File: ideas/168-paid-master-nodes/README.md [hunk 2]
```diff
@@ -156,11 +156,13 @@ technical terms on status-go side.
 Goal Date: 2018-05-07
 
 Description:
-- [ ] Recruit additional roles: Go dev, Designer, Clojure dev, QA
+- [ ] Recruit additional roles: Designer, Clojure dev (yenda?), QA
 - [ ] Spec out rough UX(R) track
```
Comment (jpbowen) [inline ideas/168-paid-master-nodes/README.md]: Can add my name here for UXR.

**Comment by pilu** [comment]
@oskarth I also fixed the YAML front matter with a simple commit here: https://github.com/status-im/ideas/pull/233

File: ideas/168-paid-master-nodes/stt-payment-testing.md (hunk 1 above)
Comment (oskarth) [reply ideas/168-paid-master-nodes/stt-payment-testing.md]: That is the contract address for STT, but it's a good point in terms of simplifying flow for testing. Captured issue here https://github.com/status-im/status-react/issues/4062

File: ideas/168-paid-master-nodes/README.md (hunk 2 above)
Comment (oskarth) [reply ideas/168-paid-master-nodes/README.md]: done

**Pull request merged**